src/javascripts/ng-admin/es6/lib/Entity/Entity.js

```diff
@@ -10,9 +10,12 @@ import ShowView from '../View/ShowView';
 import BatchDeleteView from '../View/BatchDeleteView';
 import ExportView from '../View/ExportView';
 
+var index = 0;
```

Comment:
IMO, it should be the `Application` responsability to keep track of its entities, and thus attributing them an unique id (via a `setId` method for instance). It would prevent a global variable.